redbot/core/core_commands.py

```diff
@@ -1151,15 +1154,9 @@ async def embedset_showsettings(self, ctx: commands.Context, command_name: str =
         **Arguments:**
             - `[command_name]` - Checks this command for command specific embed settings.
```

Comment:
Argument name was changed: 
```suggestion
            - `[command]` - Checks this command for command specific embed settings.
```
This also needs equivalent change in the docs.